<?xml version="1.0"?>
<launch>
  
  	<!-- node that updates calibration urdf with results -->
  	<node type="update_cob_calibration_urdf.py" pkg="cob_robot_calibration" name="$(anon update_cob_calibration_urdf)" output="screen">
  		<param name="urdf_in" value="$(find cob_calibration_data)/$(env ROBOT)/calibration/calibration.urdf.xacro" />
  		<param name="urdf_out" value="$(find cob_calibration_data)/$(env ROBOT)/calibration/calibration.urdf.xacro" />
        <param name="initial_system" value="$(find cob_robot_calibration)/config/$(env ROBOT)/system.yaml" />
        <param name="calib_system" value="/tmp/cal/result_step_3.yaml" />
        
        <!-- set to True for debug output -->
        <param name="debug" value="True" />
 	</node>
 
</launch>
